src/preset.ts

```diff
@@ -1048,6 +1048,7 @@ async function tryApplyVsDevEnv(preset: ConfigurePreset, workspaceFolder: string
                     tempPreset.environment = EnvironmentUtils.mergePreserveNull([process.env, tempPreset.environment]);
                     const expansionOpts: ExpansionOptions = await getExpansionOptions(workspaceFolder, sourceDir, tempPreset);
                     const presetEnv = tempPreset.environment;
+                    // TODO: we need to make sure we only expand relevant env vars not EVERY env var.
```

Comment:
done, could you please double check on the getExpansionOptions() call? I imagine we are not passing in anything for parent env here right? since its undefined

Reply:
Yeah `penv` here will default to process.env, which is what we want